<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CKJVPassageNavigator</class>
 <widget class="QWidget" name="CKJVPassageNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblWord">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Word:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editWord">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblVerse">
         <property name="text">
          <string>Verse:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editVerse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblChapter">
         <property name="text">
          <string>Chapter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editChapter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblBook">
         <property name="text">
          <string>Book:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editBook">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblTestament">
         <property name="text">
          <string>Bible/Testament:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboTestament">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>22</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblResolved">
         <property name="text">
          <string>Resolves to:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editResolved">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>201</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>201</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>212</red>
               <green>208</green>
               <blue>200</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
